### DISCUSSION TITLE
ZINC dataset partition

### DISCUSSION THREAD

**Discussion by wubo2180** [pyg-team/pytorch_geometric#2959] [Q&A] [answered]
Hi, I import ZINC dataset in my code. The dataset can be split into 'train','val' or 'test'. But I want to use the full dataset without splitting. Is therer any suggestion about my request? Or the dataset which is split can be merged?

**Comment by rusty1s** [comment, marked as answer]
You can wrap them inside a [`torch.utils.data.dataset.ChainDataset`](https://pytorch.org/docs/stable/_modules/torch/utils/data/dataset.html#ChainDataset).

**Reply by wubo2180** [reply]
Thanks! While the dataset is not iterable, I used [torch.utils.data.ConcatDataset](https://pytorch.org/docs/stable/_modules/torch/utils/data/dataset.html#ConcatDataset) instead.
